cachito/workers/pkg_managers/npm.py

```diff
@@ -24,6 +25,20 @@
 
 log = logging.getLogger(__name__)
 
+def _is_worksapce_version(version, workspaces):
```

Comment:
typo: workspace

we also try to use type hints when possible:
```suggestion
def _is_workspace_version(version: str, workspaces: list[str]) -> bool:
```

Reply:
:+1: